src/test/java/org/opentripplanner/updater/trip/TimetableSnapshotSourceTest.java

```diff
@@ -469,7 +469,7 @@ public void delayed() {
 
       var tripUpdateBuilder = new TripUpdateBuilder(
         tripId.getId(),
-        LocalDate.now(),
+        LocalDate.now(transitModel.getTimeZone()),
```

Comment:
This will probably create a conflict with https://github.com/opentripplanner/OpenTripPlanner/pull/4797.

If you want I can make this a draft and merge it after this one.

Reply:
I saw you fixed it as well, but the merge is easy so who ever is second can do the merge.